code/mission/missioncampaign.cpp

```diff
@@ -857,7 +857,9 @@ int mission_campaign_previous_mission()
 	}
 
 	// copy backed up containers over
-	Campaign.persistent_containers = Campaign.red_alert_containers;
+	for (const auto& ra_container : Campaign.red_alert_containers) {
+		Campaign.persistent_containers.emplace_back(ra_container);
+	}
```

Comment:
Using `std::copy()` both here and below would result in shorter code, although this version allows for more easily lining up this code with the equivalent code for persistent variables.